src/EnergyPlus/UnitarySystem.cc

```diff
@@ -233,7 +231,7 @@ namespace UnitarySystems {
                                  Real64 &sysOutputProvided,
                                  Real64 &latOutputProvided)
     {
-        int CompOn = 0;
+        DataHVACGlobals::CompressorOperation CompOn = DataHVACGlobals::CompressorOperation::Off;
```

Comment:
A `using namespace DataHVACGlobals;` statement will shorten these lines, make them more readable. Is that recommended?

Reply:
Got it. Should be a quick refactor when it happens.